src/app/components/ActiveUserLoanContainer/components/ActiveLoanTableContainer/index.tsx

```diff
@@ -37,6 +37,7 @@ export function ActiveLoanTableContainer(props: Props) {
   const [selectedItem, setSelectedItem] = useState<any>(props.data[0]);
   const [expandedItem, setExpandedItem] = useState('');
   const [expandedId, setExpandedId] = useState('');
+  const [windowSize, setWindowSize] = useState(window.innerWidth);
```

Comment:
As commented bellow in this file, it's better to use css media queries, so this is not needed

```suggestion
```